create_infotexts.py

```diff
@@ -154,9 +154,11 @@ def generate_infobox_template(item, img, places_mapping):
     infobox += "| depicted place     = " + img.data["depicted_place"] + "\n"
 
     infobox += "| date               = "
-    if not item["Fotodatum"] == "":
-        infobox += str(item["Fotodatum"])
-    infobox += "\n"
+    if item["Fotodatum"]:
```

Comment:
Yes, I think it's a good idea to add the between-template even if it's empty. It's all from that period. 
Done.